plotting.py

```diff
@@ -1301,6 +1284,48 @@ def set_missing_fieldlimits(self):
                 except ValueError:
                     self.fieldlimits[field] = [None,None]
 
+def _get_dim(df,dim,default_idx=False):
```

Comment:
Could you add a note about `default_idx`, indicating that it will return unnamed indices by default?

Reply:
Done